src/components/Burger/BuildControls/BuildControl/BuildControl.test.js

```diff
@@ -0,0 +1,72 @@
+// import React from 'react';
```

Comment:
All the test cases in the file is commented. 
This means no test cases are runnning